apps/gcd/views/search_haystack.py

```diff
@@ -127,14 +127,18 @@ def __call__(self, request, context=None):
         elif self.sort == 'year':
             self.results = self.results.order_by('year',
                                                  '-_score')
-        elif len(self.form.selected_facets) == 1:
+        elif len(self.form.selected_facets) >= 1:
             if self.sort:
-                if self.form.selected_facets[0] in \
-                  [u'facet_model_name_exact:publisher',
-                   u'facet_model_name_exact:indicia publisher',
-                   u'facet_model_name_exact:brand group',
-                   u'facet_model_name_exact:brand emblem',
-                   u'facet_model_name_exact:series']:
+                if (u'facet_model_name_exact:publisher' \
+                  in self.form.selected_facets) or \
+                  (u'facet_model_name_exact:indicia publisher'
+                  in self.form.selected_facets) or \
+                  (u'facet_model_name_exact:brand group'
+                  in self.form.selected_facets) or \
+                  (u'facet_model_name_exact:brand emblem'
+                  in self.form.selected_facets) or \
+                  (u'facet_model_name_exact:series'
+                  in self.form.selected_facets):
```

Comment:
You could put all of these facet names in a set and just test for a non-empty intersection to make this slightly more readable.  Just a thought, take or leave as you will.